src/applications/static-pages/events/components/Results/index.js

```diff
@@ -125,48 +126,11 @@ export const Results = ({
                 </div>
 
                 {/* Where */}
-                {event?.fieldLocationType === 'online' && (
-                  <div className="vads-u-display--flex vads-u-flex-direction--row vads-u-margin-top--1">
-                    <p className="vads-u-margin--0 vads-u-margin-right--0p5">
-                      <strong>Where:</strong>
-                    </p>
-
-                    <div className="vads-u-display--flex vads-u-flex-direction--column">
-                      <p className="vads-u-margin--0">
-                        This is an online event.
-                      </p>
-                    </div>
-                  </div>
-                )}
-
-                {locations?.length > 0 && (
-                  <div className="vads-u-display--flex vads-u-flex-direction--row vads-u-margin-top--1">
-                    <p className="vads-u-margin--0 vads-u-margin-right--0p5">
-                      <strong>Where:</strong>
-                    </p>
-
-                    <div className="vads-u-display--flex vads-u-flex-direction--column">
-                      {event.fieldFacilityLocation?.entity?.entityUrl?.path &&
-                        event.fieldFacilityLocation?.entity?.title && (
-                          <p className="vads-u-margin--0">
-                            <a
-                              href={
-                                event.fieldFacilityLocation.entity.entityUrl
-                                  .path
-                              }
-                            >
-                              {event.fieldFacilityLocation.entity.title}
-                            </a>
-                          </p>
-                        )}
-                      {locations?.map(location => (
-                        <p className="vads-u-margin--0" key={location}>
-                          {location}
-                        </p>
-                      ))}
-                    </div>
-                  </div>
-                )}
+                <ResultsWhereContent
+                  fieldFacilityLocation={event?.fieldFacilityLocation}
+                  fieldLocationType={event?.fieldLocationType}
+                  locations={locations}
+                />
```

Comment:
Since all of the properties here are derived from `event`, I wonder if it might be better to just define this to pass `event` and then derive the individual values inside the `ResultsWhereContent`. It might then be called `EventLocation`, perhaps.

Ex:
```
<ResultsWhereContent event={event} />
```
Or:
```
<EventLocation event={event} />
```